app/assets/stylesheets/opportunities.scss

```diff
@@ -23,6 +23,13 @@ h1, h2 {
   color: white;
 }
 
+.at-capacity {
+  background-color: #FF851B;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.